server/apps/distributed-app/docs/modules/ROOT/pages/run/run-kubernetes.adoc

```diff
@@ -0,0 +1,183 @@
+
+=== Deploy James Chart 
+
+Before doing a real deployment, you need to check that you have at least all necessary values for James to start and run smoothly on your environment. You can check the `checklist.md` in our document folder for this.
+
+Run the command when you think you are good to go:
+
+    $ helm secrets install james james/ -f james/values.yaml -f james/secrets.yaml
+
+You can also wait for the deployment to be complete by adding a --wait flag, that is based on the readiness probe located inside the James pod.
+
+However sometimes things might not always go well at the start of James. Usually it is good to look at the logs. If it goes without issues until you start to see some successful health check logs, then you are likely good.
+
+=== Upgrade Chart version
+
+If you want to deploy a new version of a installed package, use helm secrets upgrade:
+
+    $ helm secrets upgrade -f james/conf.yaml -f james/secrets.yaml james /path/to/helm/james 
+
+What Helm does is that it compares all the generated YAML files with those on the server.
+And it updates only those that changed. There are however some limitations:
+
+* pods in errors are not recreated during an upgrade. It allows to check the logs first. However, if you delete the pod by hand, it will be recreated from the last definition.
+* Some artifacts, such as jobs and stateful sets, have immutable properties. So, some upgrades may fail. Deleting by hand the element is sometimes a solution before doing the upgrade.
+
+TIP: Install link:https://github.com/databus23/helm-diff[helm-diff] plugin to display the changed configurations before run upgrade. 
+
+    $ helm secrets diff upgrade -f james/conf.yaml -f james/secrets.yaml james /path/to/helm/james 
+
+=== Rollback to previous Chart version
+Show chart revision history:
+
+    $  helm history james -n <name space>
+
+Rollback to a previous version:
+
+    $  helm rollbaack james <revision>
+
+=== Uninstalling
+To uninstall the helm package:
+
+    $ helm uninstall james -n <name space>
+
+== References
+
+A summary of different resources that could be interesting to have in hand.
+
+=== This package
+
+* xref:run/k8s-values.adoc[Values]
+* xref:run/k8s-secrets.adoc[Secrets]
+* xref:run/k8s-checklist.adoc[Check List]
+* xref:run/k8s-logsMetrics.adoc[Logs and Metrics]
+
+### James documentation
+
+* link:https://james.apache.org/[James Official Website]
+* link:https://github.com/apache/james-project[James Github Project]
+* link:https://james.staged.apache.org/james-distributed-app/3.7.0/operate/webadmin.html[Webadmin]
+* link:https://github.com/apache/james-project/blob/master/src/site/markdown/server/manage-cli.md[James CLI]
+* link:https://github.com/apache/james-project/blob/master/src/site/markdown/server/manage-guice-distributed-james.md[Manage Guice Distributed James]
```

Comment:
These 3 links should be xrefs to the current documentation

Reply:
Fixed with james.apache.org link:
```
* link:https://james.apache.org/server/manage-webadmin.html[WebAdmin]
* link:https://james.apache.org/server/manage-cli.html[James CLI]
* link:https://james.apache.org/server/manage-guice-distributed-james.html[Manage Guice Distributed James]
```

Reply:
xref:operate/cli.adoc[...] is better

Reply:
the `Manage Guice Distributed James` is a md in `src/site/markdown/server/manage-guice-distributed-james.md`